addon/components/frost-button.js

```diff
@@ -227,12 +246,17 @@ export default Component.extend(PropTypeMixin, {
     }
   }),
 
+  // FIXME: jsdoc
   _onFocus: Ember.on('focusIn', function (e) {
     // If an onFocus handler is defined, call it
     if (this.attrs.onFocus) {
       this.attrs.onFocus()
     }
-  })
+  }),
+
+  // == Lifecycle Hooks =======================================================
 
   // == Actions ===============================================================
+
+  actions: {}
```

Comment:
B/c it lets you always have a trailing comma on methods above it, instead of having to worry about having one or not depending on if it's currently the last one.